lib/AST/ASTMutations.cpp

```diff
@@ -0,0 +1,72 @@
+#include "mull/AST/ASTMutations.h"
+
+#include <llvm/Support/FileSystem.h>
+
+#include <iostream>
+
+using namespace mull;
+
+ASTMutations::ASTMutations() : storage() {}
+
+int ASTMutations::count() const {
+  int count = 0;
+  for (auto &pair : storage) {
+    /// std::cout << "ASTMutations.count() one ast unit: " << pair.first << "\n";
+    count += storage.at(pair.first).size();
+  }
+  return count;
+};
+
+bool ASTMutations::locationExists(SourceFilePath sourceFile, int line,
+                                  int column) const {
+  assert(llvm::sys::fs::is_regular_file(sourceFile) ||
+         sourceFile == "input.cc");
```

Comment:
"input.cc" is what you get by default when you create AST from a string using clang tooling.

I haven't found a better way to skip `input.cc` but at the same time assert that the paths are real when running real `mull-cxx`.